src/test/regress/expected/multi_create_fdw.out

```diff
@@ -9,3 +9,71 @@ AS 'citus'
 LANGUAGE C STRICT;
 CREATE FOREIGN DATA WRAPPER fake_fdw HANDLER fake_fdw_handler;
 CREATE SERVER fake_fdw_server FOREIGN DATA WRAPPER fake_fdw;
+-- test propagating foreign server creation
+CREATE EXTENSION postgres_fdw;
+CREATE SERVER foreign_server TYPE 'test_type' VERSION 'v1'
```

Comment:
I think we should have a very basic test with `citus_add_node`. Probably here is not the best place to add such a test, but can you find a place where we ensure that adding node sends the foreign server to the new node as well?

Reply:
I went over the test schedule but this file seemed like the most relevant place for this test. So I've added it to `multi_create_fdw.sql`.

If you have a suggestion I could consider moving it